### PR TITLE
Describe definition of ready for 3 kinds of tasks

### PR DESCRIPTION
**Pull request by yuvipanda**
Ref https://github.com/2i2c-org/meta/issues/953

### CODE REVIEW

File: engineering/definition-of-ready.md [hunk 1]
```diff
@@ -51,16 +51,37 @@ should not be considered a feature we currently support!
...
+
+- [ ] Link to documentation about this feature, which has at least some mention of the possible
+      ways to make configuration changes. Since 2i2c documentation may not cover all the possible
+      options (as that would be too unweildy), this could be either a link to 2i2c documentation on
+      [infrastructure.2i2c.org](https://infrastructure.2i2c.org) or a link to upstream documentation
+      about various configuration options for this feature.
+- [ ] Any additional information needed for this specific configuration change.
+
+Functionally, this means that if we are requested to change config for something that we can't
+find any documentation for, we should
```
Comment (consideRatio) [inline engineering/definition-of-ready.md:68]: Unfinished sentence

**Review by GeorgianaElena — APPROVED** [review]
LGTM

File: engineering/definition-of-ready.md (hunk 1 above)
Comment (yuvipanda) [reply engineering/definition-of-ready.md:68]: Whoops. I completed this sentence, lmk what you think

**Comment by consideRatio** [comment]
This is tricky to be specific about! Not sure about an action point, but reflected a bit.

1. A config change not documented yet could blur the line of new feature.
2. A config change could be simple and easy, but perhaps involve unsustainable toil, unfair in a shared cluster with regards to sharing costs, or have some reason making it worth deciding on if we should or shouldn't do something like this for communities.

I think its good that this is escalated to #engineering to be decided about by more than one person.

Documenting likely makes this work take more than 30 minutes, so what should the triager do?
- schedule to be done in s sprint
- config change now+document now
- config change now + create an issue to document
- document now + create an issue to config change

Maybe to be decided in engineering slack chat as well?

**Review by consideRatio — APPROVED** [review]

**Review by sgibson91 — APPROVED** [review]

**Pull request merged**